### ISSUE TITLE
Diff for months results in negative zero when no difference.

### ISSUE THREAD

**Issue by archemiro** [iamkun/dayjs#386]
In momentjs I had a months comparison of dates, when the difference is zero months, the result was `0`. 

With dayjs the result is `-0`, which breaks test cases using something like `expect(diff).toBe(0)` or `expect(diff).toEqual(0)`

```js
var moment = require("moment")
var dayjs = require("dayjs")

var momentDiff = moment("2018-01-01").diff("2018-01-01", "months")
var dayjsDiff = dayjs("2018-01-01").diff("2018-01-01", "months")

console.log(momentDiff) // 0
console.log(dayjsDiff) // -0
```

**Comment by suspectpart** [comment]
It also returns `-0` for unit `'years'`, does the fix cover that as well?

**Comment by iamkun** [comment]
fix #390
